CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+
+Contributing code
+=================
+
+How to contribute
+-----------------
+
+The preferred way to contribute to Kymatio is to fork the
+[main repository](http://github.com/kymatio/kymatio/) on
+GitHub:
+
+1. Fork the [project repository](http://github.com/kymatio/kymatio):
+   click on the 'Fork' button near the top of the page. This creates
+   a copy of the code under your account on the GitHub server.
+
+2. Clone this copy to your local disk:
+
+          $ git clone git@github.com:YourLogin/kymatio.git
+          $ cd kymatio
+
+3. Remove any previously installed versions of Kymatio:
+
+          $ pip uninstall kymatio
+
+and install your local copy with testing dependencies:
+
+          $ pip install -e .[tests]
```

Comment:
this is incorrect and needs a fix

Reply:
No worry, what is the goodway?

Reply:
i'm making a 2nd pr to your branch re: contributing.md

Reply:
see https://github.com/edouardoyallon/kymatio/pull/2